src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -104,6 +122,7 @@ static void usage()
             << "  --exclusive             Forbid writes by other clients\n"
             << "  --max_part <limit>      Override for module param max_part\n"
             << "  --nbds_max <limit>      Override for module param nbds_max\n"
+            << "  --reconnect             Reconnect to recently detached device\n"
```

Comment:
Nit: just thinking, but would it perhaps make more sense to have a `map` command and a separate `attach` command to be more explicit with the option parsing? That would eventually allow support for a similar `unmap` vs `detach` command.

Reply:
I think I like additional `attach` + `detach` commands. Do we also want `rbd device attach|detach` wrappers? I think we could decide and add this later if it proves to be useful.

Would it make sense for `detach` to wait for the process to actually terminate? This would allow to avoid races when `detach` following by `attach` are executed.

Reply:
> Do we also want `rbd device attach|detach` wrappers? I think we could decide and add this later if it proves to be useful.

I'm OK with waiting to add then to the "rbd" CLI -- it's a pretty narrow use-case.

> Would it make sense for `detach` to wait for the process to actually terminate? This would allow to avoid races when `detach` following by `attach` are executed.

If we can get "rbd-nbd" to properly wait for the `unmap` and `detach` cases, I think that would be great!